src/itemTracker/SwordBlock.tsx

```diff
@@ -13,6 +13,14 @@ export default function SwordBlock({ width }: { width: number }) {
         dispatch(clickItem({ item: 'Extra Wallet', take: false }));
     };
 
+    const handleExtraWalletDrag = (event: React.DragEvent<HTMLDivElement>) => {
+        event.dataTransfer.setData('text/plain', 'Extra Wallet');
+        event.dataTransfer.effectAllowed = 'move';
+        const dragIcon = new Image(36, 36);
+        dragIcon.src = findRepresentativeIcon('Extra Wallet');
+        event.dataTransfer.setDragImage(dragIcon, 18, 18);
```

Comment:
This only works the second time someone attempts to drag an extra wallet since it's not loaded the first time.